soc/arm/microchip_mec/Kconfig

```diff
@@ -11,6 +12,210 @@ config SOC_FAMILY
 	string
 	default "microchip_mec"
 
+menuconfig MCHP_MEC_HEADER
+	bool "The output binary with MCHP MEC binary header"
```

Comment:
"Create an unsigned output binary with the MCP MEC binary header"

Reply:
ok